master/internal/api_tasks.go

```diff
@@ -329,19 +329,31 @@ func (a *apiServer) GetTaskAcceleratorData(
 		return nil, err
 	}
 
-	var res []model.AcceleratorData
-	err := db.Bun().NewSelect().
-		ColumnExpr("alloc_acc.*").
+	res := []struct {
+		model.AcceleratorData
+		ResourcePool string
+	}{}
+
+	err := db.Bun().NewSelect().ColumnExpr("alloc_acc.*").
```

Comment:
can you update a test in ```api_tasks_intg_test.go``` to check resource pool is set right?

Reply:
Of course, test added